### PR TITLE
Make Clay a Pebble package

### LINKED ISSUE #97: Clay pebble-package
Will you create a clay pebble-package for an easier integration in our projects ? :-p

Thanks !

### PR DESCRIPTION
**Pull request by keegan-lillo**
resolves #97

### CODE REVIEW

**Comment by keegan-lillo** [comment]
@cjwilliams can I get you please to check my new updated getting started?

File: README.md [hunk 1]
```diff
@@ -17,23 +17,22 @@ Prior to SDK 3.13, `require` paths were handled in a non-standard way. When requ
...
 
   ```javascript
-  var Clay = require('./clay');
+  var Clay = require('pebble-clay');
   var clayConfig = require('./config.json');
   var clay = new Clay(clayConfig);
   ```
-6. Add `configurable` to the `capabilities` array in your `appinfo.json`.
-7. Ensure `enableMultiJS` is set to true in your `appinfo.json`.
+6. Add `configurable` to the `pebble.capabilities` array in your `package.json`.
```
Comment (cjwilliams) [inline README.md]: This shouldn't be necessary. If capabilities are present in the pebble package's package.json file, they will automatically be merged with the capabilities of the app using the pebble package.

File: README.md [hunk 2]
```diff
@@ -17,23 +17,22 @@ Prior to SDK 3.13, `require` paths were handled in a non-standard way. When requ
...
+7. Ensure `pebble.enableMultiJS` is set to true in your `package.json`.
 8. Next is the fun part - creating your config page. Edit your `config.js` file to build a layout of elements as described in the sections below.
```
Comment (cjwilliams) [inline README.md]: config.js or config.json?

File: README.md (hunk 1 above)
Comment (keegan-lillo) [inline README.md]: Oh cool. DIdn't know that.

File: README.md [hunk 3]
```diff
@@ -137,7 +135,7 @@ Headings can be used in anywhere and can have their size adjusted to suit the co
 |----------|------|-------------|
 | type | string | Set to `heading`. |
 | id | string (unique) | Set this to a unique string to allow this item to be looked up using `Clay.getItemsById()` in your [custom function](#custom-function). |
-| appKey | string (unique) | The AppMessage key matching the `appKey` item defined in your `appinfo.json`.  Set this to a unique string to allow this item to be looked up using `Clay.getItemsByAppKey()` in your custom function. You must set this if you wish for the value of this item to be persisted after the user closes the config page. |
+| appKey | string (unique) | The AppMessage key matching the `appKey` item defined in your `package.json`.  Set this to a unique string to allow this item to be looked up using `Clay.getItemsByAppKey()` in your custom function. You must set this if you wish for the value of this item to be persisted after the user closes the config page. |
```
Comment (cjwilliams) [inline README.md]: Not to be confusing, but `appKeys` exist in appinfo.json, `messageKeys` exist in package.json, and other variations result in a build error.

**Comment by cjwilliams** [comment]
A few nits, but otherwise looks good. 👍

**Pull request merged**